src/components/YieldPools/index.tsx

```diff
@@ -119,8 +121,8 @@ const YieldPools = ({ loading, active }: { loading: boolean; active?: boolean })
   const noFarms = !Object.keys(farms).length
 
   useEffect(() => {
-    setSearchText('')
-  }, [active])
+    setSearchText(search ?? '')
```

Comment:
why do you need setSearchText here? fully use query param please
(push search string to url when user search)

Reply:
https://github.com/KyberNetwork/dmm-interface/blob/add-liquid-pro-amm/src/pages/Pools/index.tsx#L47..L51
An example for you

Reply:
Fixed that @viet-nv